source/_drafts/Feature-Spotlight-Middleware.md

```diff
@@ -0,0 +1,185 @@
+---
+title: 'Feature Spotlight: Middleware'
+author:
+  name: Gary Katsevman
+  github: gkatsev
+tags:
+  - middleware
+  - video.js 6.0
+---
+
+Middleware is one of the cool new features that is coming to Video.js in version 6.0
+
+With middleware, you are now able to interact with and change how the player and the tech talk to each other.
+The tech is Video.js's abstraction from the player. Separating the player API from the playback technology. With techs, we can plug things like a Flash fallback or a Youtube embed into Video.js
```

Comment:
"_... from the player. Separating the player API ..._" should be "_... from the player, separating the player API ..._". And I'd suggest "... or a _YouTube_ embed into Video.js _without changing the external API or the look-and-feel of the player_".